api/handler/default_test.go

```diff
@@ -21,6 +20,7 @@ import (
 	"github.com/ordishs/go-bitcoin"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"k8s.io/utils/ptr"
```

Comment:
not very big fan of adding dependency just for 3 lines. 
I'd suggest just adding following method instead:
```
func To[T any](v T) *T {
	return &v
}
```
If we end up using more functions from this lib then we could probably refactor. Wdyt?